tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -331,6 +331,18 @@ def disable_asserts(input_string):
         output_string = output_string.replace(input_string[start:p_end + 1], "")
     return output_string
 
+def replace_forceinline(input_string):
```

Comment:
Why is this function needed?

Reply:
@jithunnair-amd could you add a doc string here with the link to HIP not implying static for forceinline?

Reply:
@iotamudelta please don't do this, the correct fix is to change how HIP handles `__forceinline__` to match what CUDA does (it actually implies language `inline` there). Please ask the HIP maintainers to correct that.

Reply:
@AlexVlx yes, we will. for the time being we need things to compile reliably in all our environments. it's a hack, unfortunately. @jithunnair-amd could you create HIP issue w/ that and link it here for documentation purposes? thanks! :-)

Reply:
https://github.com/ROCm-Developer-Tools/HIP/issues/572